scripts/dynamicHtml.js

```diff
@@ -154,13 +154,14 @@ function newImageUpload(uid, {fnName = "", active = true} = {}) {
         <button class="btn btn-info font-weight-bold" id="viewFinalImgBtn_${uid}">Converted</button>
       </div>
     </div>
-    <div class="btn-group">
-      <button class="btn btn-info" id="saveFormDataBtn_${uid}">💾 Save
-      <a data-toggle="tooltip" data-placement="top" data-html="true" title="Save to the browser's local storage.
-        No data is sent to the server. Has limited storage capacity. Delete other images if you cannot save anymore."
-        data-delay="{&quot;show&quot;:100, &quot;hide&quot;:2000}"
-        class="text-info text-info-inverted">${icons.questionmark}</a></button>
-      <button class="btn btn-warning mr-sm-3" id="imgEditBtn_${uid}">Edit</button>
+    <div>
+      <button class="btn btn-info" id="saveFormDataBtn_${uid}">${icons.save} Save
+        <a data-toggle="tooltip" data-placement="top" data-html="true" title="Save to the browser's local storage.
+          No data is sent to the server. This form will be restored the next time you visit this site."
```

Comment:
Felt that the explanation related to deletion / insufficient storage can be shown in the error message only, in case of failure - it simplifies the info here.

Reply:
Also replaced the emojis with more minimalistic SVG icons to keep the style in sync with icons at other places in the site.

And if emojis don't render due to outdated system/OS fonts, it can be confusing 
<img width="149" alt="Screenshot 2024-03-05 at 23 11 52" src="https://github.com/gd-codes/mc-pixelart-maker/assets/45459312/2c234ac9-e09f-4d87-8a70-43695c9e8c20">

Reply:
Good point regarding the tooltip. I created it before I added the error message. 👍
As for the emoji, it worked for me. ;-)
I find the icons in the tab headers a bit bland, since they have the same color as the text, i.e. black or blue. Maybe I'm biased because I used the colorful emojis for a few days now. :-)

Reply:
For reference, this is how the tab headers look for me now:
![image](https://github.com/gd-codes/mc-pixelart-maker/assets/6648929/227db847-8a72-416c-a56c-b3dd3fecaef1)